pkg/antctl/antctl.go

```diff
@@ -81,6 +82,36 @@ var CommandList = &commandList{
 			},
 			transformedResponse: reflect.TypeOf(version.Response{}),
 		},
+		{
+			use:   "podmulticaststats",
+			short: "Show multicast statistics",
+			long:  "Show multicast traffic statistics of ${component}",
```

Comment:
```suggestion
			long:  "Show multicast traffic statistics of Pods",
```
I think the stats are Pods', not agent's